torch/csrc/jit/codegen/cuda/scheduler/pointwise.cpp

```diff
@@ -67,32 +67,20 @@ c10::optional<PointwiseParams> getPointwiseHeuristics(
   if (TensorDomain::noReductions(
           TensorDomain::noBroadcasts(largest_out->domain()->domain()))
           .size() == 0) {
-    // Create empty entries for vectorizable inputs outputs
-    //  and mapping count
     auto vectorizable_inputs_outputs_entry = HeuristicSummaryEntry<
```

Comment:
Probably missed this last time. Should we return early under this case? 

These empty entries will be overwritten by line 145 if there's no early return. Or maybe we could change `HeuristicSummaryCache` to `write once` behavior.

Reply:
I think we should be able to just return early, checking if there's any failing tests from this.